docs/index.rst

```diff
@@ -17,7 +17,7 @@ To clone the current main (development) branch run:
 
 .. code::
 
-    git clone git://github.com/sosreport/sos.git
+    git clone git@github.com:sosreport/sos.git
```

Comment:
would it be better to have the http endpoint, that way people don't have to specifically setup ssh? Especially if people don't have GH setup?

Reply:
IIRC doesn't GH have plans in play to drop non-authenticated http? Meaning someone would have to do some setup/configuration anyways (and GH strongly encourages the use of ssh)?